### ISSUE TITLE
Improve / Export for linux users

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-gui#1157]
After discussions on #1151 with @ehlers I see the limit of the current import / export (it's good we already got feedbacks :) ) 

The issue is Linux users could run Docker/Qemu/IOU on their local computer and the topology layout will be not compatible. 

Perhaps a solution. 

1) During the import we merge all VM type in the same directory. So instead of:

```
project-files/
    dynamips/
        server1
servers/
    vm/
       iou/
         startup-config.cfg
```

We get:
```
project-files/
    dynamips/
        server1
    iou/
       startup-config.cfg
```

2) At import the process will choose on which server the process should copy data depending of what the server support. With a naive implementation at start:
* dynamips, vpcs = local server
* if windows the others run on the GNS3 VM
* if Linux IOU it's depend if the GNS3 VM is enable
* if Linux docker & qemu use the local server

Problem is when people want to use ESXi :(

It seem more portable in the case of the user want to scale the topology on multiple servers in the futur.

**Comment by ehlers** [comment]
I would make the decision, if local server or not, more dependent of the "Use the local server" setting in the preference. In most cases that should give you the right answer, if you should choose local or remote server. So even on windows, if the user has unticked the "Use the local server" setting for dynamips the GNS3 VM will be used.

If the user uses several remote servers, you should look into the VM preference for which server the user has defined that VM. If there are several or none, the user must decide.

To sum up, I would mostly use the already defined settings for the decision, what server should be used.

**Comment by ehlers** [comment]
What about the following algorithm? Here the GNS3-VM is a remote server like any other.
- vpcs: If "Use the local server" is enabled use the local server, else use a remote server.
- For non-vpcs lookup of the image in the preference. use the server that's defined for that image.

If no server can be used, generate an error. If several server are possible, ask the user which server he wants to use for this image. Remember that answer for other devices with the same image.

**Comment by noplay** [comment]
Actually the use local server doesn't exits for Qemu, Docker and IOU.

Query for image could work but it seem complex. For 1.5 I think we will not do it because with 2.0 architecture we will need to rewrite all of this. But the idea is interesting I didn't think about that.

**Comment by noplay** [comment]
My proposal for 1.5 we support two scenario for Linux:
* nothing on the GNS3 VM
* everything on the GNS3 VM
